File: Easy/problem58.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        i = len(s)-1
        res = ""
        if len(s) == 1: 
            print(1)
            return len(s)

        while i >= 0:
            if s[i] == ' ':
                print(2)
                i-=1
            else:
                break
        
        while i>=0:
            if s[i] == ' ':
                print(3)
                break
            else:
                print(4)
                res += s[i]
                i-=1
        
        if i == 0:
            if len(res) == 0:
                return len(s[0])

        return len(res)

File: Easy/problem58.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int count = 0;
        for (int i = s.length()-1; i >=0; i--) {
            if (s[i] == ' ') {
                if (count <= 0) {
                    continue;
                } else {
                    return count;
                }
            }
            else {
                count++;
            }
        }
            return count;
    }
};
